FastpackTranspiler/Class.ml

```diff
@@ -244,20 +246,75 @@ let transpile {Context. require_runtime; _ } program =
     | _ -> (loc, node)
   in
 
-  let map_statement _scope ((loc, stmt) : Loc.t S.t) =
+  let map_statement {AstMapper. parents; _ } ((loc, stmt) : Loc.t S.t) =
     match stmt with
-    | S.ClassDeclaration cls ->
+    | S.ExportDefaultDeclaration ({
+        declaration = S.ExportDefaultDeclaration.Declaration (
+            _,
+            S.ClassDeclaration ({ id = Some (_, name); _ } as cls)
+          );
+        _
+      } as export) ->
       begin
         match Transform.transform_class cls with
         | cls, _, [], [], [] ->
-          (loc, S.ClassDeclaration cls)
-        | cls, Some (_, name), statics, classDecorators, decorators ->
+          [
+            loc,
+            S.ExportDefaultDeclaration {
+              export with
+              declaration = S.ExportDefaultDeclaration.Declaration (
+                  Loc.none,
+                  S.ClassDeclaration cls
+                )
+            }
```

Comment:
why curly brackets are aligned, but parentheses shifted?

Reply:
In fact, I used `ocp-indent` this time around. I this we should convert all the code to use it at a certain point

Reply:
Just a bit weird and inconsistent from my point of view.

Reply:
Well, I tend to agree with you, and in fact, have been ignoring the `ocp-indent` so far. But, to be honest, I think we should be adopting some automated code-styling. At the moment, I see the `ocp-indent` as the less intrusive option comparing to the `ocamlformat`

Reply:
Is there an option for this in ocp-indent?